packages/sdk-components-react-radix/src/theme/tailwind-classes.ts

```diff
@@ -843,6 +850,15 @@ export const transition = (value: "none" | "all"): EmbedTemplateStyleDecl[] => {
   ];
 };
 
+export const duration = (ms: number): EmbedTemplateStyleDecl[] => {
```

Comment:
what do we do with these unparsed values once we actually have a transition duration that is parsed and such? Isn't it easier to already add a data type?

Reply:
Style panel is able to parse these

Reply:
but do you want this data structure long term in the db? otherwise whats the point in all the data structures we have, they all could be unparsed?

Reply:
Migrations maybe
Cc @istarkov

Reply:
Isn't it just unit?